### DISCUSSION TITLE
Q about -A and disabling decoders

### DISCUSSION THREAD

**Discussion by pconroy328** [merbanan/rtl_433#2510] [Q&A] [answered]
I have a question please, about decoding an unknown device. I picked up a GUTA branded TPMS and am trying to see if I can figure out the protocol.

I believe it’s on 433.92MHz, but only because their repeater mentions that frequency on the Amazon page.

I have two other 433MHz devices in the car – a Nexus and Acurite temp sensors.  rtl_433 picks them up and displays the data beautifully!

What I thought would be a good idea would be to disable the Nexus and Acurite decoders, and then run with -A:  rtl_433 -R -19 -R -11 -R -55 -A

My question is, will the output from -A now have the Nexus and Acurite filtered out?  Will the output of -A just be, what I presume to be, the TPMS system?

**Comment by zuckschwerdt** [comment, marked as answer]
The decoder options only enable/disable decoders not the reception. The analyzer will always see all transmissions.
But if you leave the decoders enabled and opt to save transmissions with `-S unknown` you'll likely only get the TPMS as saved files.
Then run `-A` on each file. That way you can inspect the transmissions later and try things.

**Reply by pconroy328** [reply]
Thank you! That makes sense.  I'll try that
